scripts/local_install_linux.sh

```diff
@@ -9,5 +9,5 @@ set -e
 
 mkdir -p $HOME/.perlang/nightly/bin
 
-dotnet publish Perlang.ConsoleApp/Perlang.ConsoleApp.csproj -c Release -r linux-x64 --self-contained true /p:PublishReadyToRun=true /p:SolutionDir=$(pwd)/
-cp -r Perlang.ConsoleApp/bin/Release/net5.0/linux-x64/publish/* $HOME/.perlang/nightly/bin
+dotnet publish src/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj -c Release -r linux-x64 --self-contained true /p:PublishReadyToRun=true /p:SolutionDir=$(pwd)/
+cp -r src/Perlang.ConsoleApp/bin/Release/net5.0/linux-x64/publish/* $HOME/.perlang/nightly/bin
```

Comment:
(Drive-by bug fix for something which was forgotten in #167. One new idea I had while reviewing this: it would perhaps be worth running this script in CI, to prevent bugs like this from slipping through.)